projects/fizzbuzz/python/LoganShy/README.md

```diff
@@ -0,0 +1,14 @@
+Thank you for selecting my fine program, and I'm hopeful that you'll find things satisfactory 
+concerning the operation of this software. Below are some instructions that you may/may not find useful.
+
+INSTRUCTIONS:
+
+ALL USERS:
+1. If not installed, head to https://www.python.org/downloads/ and select your version to download python.
+2. If possible, when prompted to, add python to your local PATH.
+3. Save the fizzbuzz.py program located in this directory somewhere on your local machine.
+4. Try to double click the fizzbuzz py file, it should open and run. If not, proceed to step 5.
```

Comment:
could probably remove this step and only give instructions that you know will work like the terminal